lib/data.rb

```diff
@@ -0,0 +1,67 @@
+require './lib/dmv'
```

Comment:
We only need these require statements in the test file. 
Also, convention is that the name of the file should match the class name, so this file should be create.rb